lib/modules.nix

```diff
@@ -1146,14 +1141,11 @@ let
     use = id;
   };
 
-  /* Transitional version of mkAliasOptionModule that uses MD docs. */
-  mkAliasOptionModuleMD = from: to: doRename {
-    inherit from to;
-    visible = true;
-    warn = false;
-    use = id;
-    markdown = true;
-  };
+  /* Transitional version of mkAliasOptionModule that uses MD docs.
+
+     This function is no longer necessary and merely an alias of `mkAliasOptionModule`.
+  */
+  mkAliasOptionModuleMD = mkAliasOptionModule;
```

Comment:
Should `mkAliasOptionModuleMD` be marked as deprecated? I can't find information about deprecating functions in the manual, but maybe a simple wrapper which prints a deprecation message could be used here?

Ditto for the other no-op aliases.

Reply:
There's only ~10 calls to `mkAliasOptionModuleMD` in nixpkgs master. I'd say ideally we'd just replace all calls in-tree with `mkAliasOptionModule` in this PR, and then do another PR to remove `mkAliasOptionModuleMD` itself. Otherwise this deprecated function could stay around for a long time until someone takes the time to clean up.

Reply:
> keeping the alias doesn't hurt while we're not sure whether it's used anywhere else

I wasn't suggesting removing the alias in this PR, just calling `mkAliasOptionModule` instead of `mkAliasOptionModuleMD` everywhere.

Reply:
> we can't add loud deprecation warnings to any of these public apis before all supported nixos versions agree on what the documentation language _is_, otherwise we'll risk breaking out-of-tree modules, backports (official in nixpkgs and unofficial by users), and who knows what else. this applies to all functions that are now aliases or no-ops.

I think something must've been lost in translation, because my latest comment had nothing to do with deprecation. Basically just `sed -i -e s/mkAliasOptionModuleMD/mkAliasOptionModule` in all files except for the `mkAliasOptionModuleMD = mkAliasOptionModule;` line itself. This changes no functionality anywhere, doesn't introduce any deprecation warnings, and makes it pretty obvious that we can remove `mkAliasOptionModuleMD` ASAP.
 
> refactorings in nixpkgs modules are less critical, but we're not going to do any. out-of-tree modules wouldn't be affected by this, but backports would be, and we're frankly sick of refactoring docs. if you want to see this done you're totally free to open a follow-up pr once this is done, but we won't be doing it.

Fair enough.